xvm/compile/compile.go

```diff
@@ -85,8 +85,8 @@ func CompileNativeLibrary(cfg *Config, target, source string) error {
 		"-o"+target,
 		"-I.",
 		"-I"+tmpdir,
-		"-lm",
 		csource,
+		"-lm",
```

Comment:
So you put the option "-lm" in the end ? why it works

Reply:
> So you put the option "-lm" in the end ? why it works

csource needs math library.